MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -679,14 +679,19 @@ - (void)closeApp __attribute__((noreturn)) {
   exit(0);
 }
 
-- (void)openUrl:(NSURL *)url {
+- (BOOL)openUrl:(NSURL *)url {
+  
+  /*
+   * Ignore if app secret not set, can't test scheme.
+   * Also ignore if request is not for Mobile Center Distribute and this app.
+   */
+  if (!self.appSecret || ![[url scheme] isEqualToString:[NSString stringWithFormat:kMSDefaultCustomSchemeFormat, self.appSecret]]) {
```

Comment:
I am a bit divided  here. We could also just check for "mobilecenter" at the beginning of the scheme but in this case it just validate whereas the scheme is for mobile center but not specifically for this app.
Any suggestions?